### PR TITLE
Bump FSharp.Core to 4.3.0

### PR DESCRIPTION
**Pull request by nosami**

### CODE REVIEW

**Comment by nosami** [comment]
small whitespace _improvements_ :)

File: main/external/fsharpbinding/MonoDevelop.FSharp.Tests/MonoDevelop.FSharp.Tests.fsproj [hunk 1]
```diff
@@ -156,10 +113,14 @@
...
-    <ProjectReference Include="..\MonoDevelop.FSharp.Shared\MonoDevelop.FSharp.Shared.fsproj">
-      <Project>{AF5FEAD5-B50E-4F07-A274-32F23D5C504D}</Project>
-      <Name>MonoDevelop.FSharp.Shared</Name>
-    </ProjectReference>
+    <Content Include="Samples\android\fsandroidnuget2\MainActivity.fs">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="Samples\android\packages\FSharp.Core.4.0.0.1\lib\portable-net45+monoandroid10+monotouch10+xamarinios10\FSharp.Core.dll">
```
Comment (dsyme) [inline main/external/fsharpbinding/MonoDevelop.FSharp.Tests/MonoDevelop.FSharp.Tests.fsproj]: Surprised to see 4.0.0.1 mentioned when the TargetFSharpCoreVersion has been bumped to 4.4.3.0 (consistent with nuget packages 4.3.0-4.3.3)

File: main/external/fsharpbinding/MonoDevelop.FSharpBinding/MonoDevelop.FSharp.fsproj [hunk 2]
```diff
@@ -58,7 +58,7 @@
     <ItemGroup>
       <NuGetPackage Include="FSharp.Core">
         <Name>fsharp.core</Name>
-        <Version>4.1.0.2</Version>
+        <Version>4.3.0</Version>
```
Comment (dsyme) [inline main/external/fsharpbinding/MonoDevelop.FSharpBinding/MonoDevelop.FSharp.fsproj]: Could you make this 4.3.3 please?

File: main/external/fsharpbinding/MonoDevelop.FSharpBinding/Templates/FSharpGtkProject.xpt.xml [hunk 3]
```diff
@@ -37,7 +37,7 @@
 			</References>
 
       <Packages requireLicenseAcceptance="false">
-        <Package Id="FSharp.Core" Version="4.1.0.2" local="true" /> 
+        <Package Id="FSharp.Core" Version="4.3.0" local="true" />
```
Comment (dsyme) [inline main/external/fsharpbinding/MonoDevelop.FSharpBinding/Templates/FSharpGtkProject.xpt.xml]: likewise here and elsewhere

File: main/external/fsharpbinding/MonoDevelop.FSharp.Tests/MonoDevelop.FSharp.Tests.fsproj (hunk 1 above)
Comment (nosami) [reply main/external/fsharpbinding/MonoDevelop.FSharp.Tests/MonoDevelop.FSharp.Tests.fsproj]: this one is OK I think, just a sample project used by a test

File: main/external/fsharpbinding/MonoDevelop.FSharpBinding/MonoDevelop.FSharp.fsproj (hunk 2 above)
Comment (nosami) [reply main/external/fsharpbinding/MonoDevelop.FSharpBinding/MonoDevelop.FSharp.fsproj]: sure

**Comment by dsyme** [comment]
Good job. added a couple of comments - main one is to use nuget package 4.3.3 instead of 4.3.0

**Pull request merged**